relay-general/src/pii/snapshots/relay_general__pii__processor__tests__scrub_original_value.snap

```diff
@@ -0,0 +1,180 @@
+---
+source: relay-general/src/pii/processor.rs
+expression: "&data"
+---
+Event {
+    id: ~,
+    level: ~,
+    version: ~,
+    ty: ~,
+    fingerprint: ~,
+    culprit: ~,
+    transaction: ~,
+    transaction_info: ~,
+    time_spent: ~,
+    logentry: ~,
+    logger: ~,
+    modules: ~,
+    platform: ~,
+    timestamp: ~,
+    start_timestamp: ~,
+    received: ~,
+    server_name: ~,
+    release: ~,
+    dist: ~,
+    environment: ~,
+    site: ~,
+    user: User {
+        id: ~,
+        email: ~,
+        ip_address: Meta {
+            remarks: [],
+            errors: [
+                Error {
+                    kind: InvalidData,
+                    data: {
+                        "reason": String(
+                            "expected an ip address",
+                        ),
+                    },
+                },
+            ],
+            original_length: None,
+            original_value: None,
+        },
+        username: Annotated(
+            "hey  man [ip]",
+            Meta {
+                remarks: [
+                    Remark {
+                        ty: Substituted,
+                        rule_id: "@ip:replace",
+                        range: Some(
+                            (
+                                9,
+                                13,
+                            ),
+                        ),
+                    },
+                ],
+                errors: [],
+                original_length: Some(
+                    22,
+                ),
+                original_value: None,
```

Comment:
previously the original value would be leaked here. now since IpAddr is set to scrub anything it gets removed completely. if it's a value that is set to only be scrubbed partially you might see something like `" [ip]"`